scripts/genpinctrl/stm32-pinctrl-config.yaml

```diff
@@ -11,10 +11,12 @@
 #     be as precise as possible. Note that it needs to be escaped here in the
 #     configuration file.
 #
-#   - mode (mandatory): Mode setting (analog, alternate). Mode needs to
+#   - mode (optional): Mode setting (analog, alternate). Mode needs to
 #     be set according to the following rules:
 #       * Pin operates in analog configuration: analog
 #       * Pin operates in alternate function configuration: alternate
+#     In default cases, mode could be infered from inputs files and could be
```

Comment:
In default `case`, mode `can` be `inferred` from `CubeMX configuration files` and `can` be omitted.

Reply:
Fixed